frontend/00-browsertrix-nginx-init.sh

```diff
@@ -12,13 +12,13 @@ else
   sed -i "s/\$LOCAL_BUCKET/$LOCAL_BUCKET/g" /etc/nginx/includes/minio.conf
 fi
 
-# Plausible analytics
+# Add analytics script, if provided
```

Comment:
Maybe we wanna just call this "head_extra" or something like that?

Reply:
And rename to to extra.js? sure, that can work